i18n/fr/system76_keyboard_configurator.ftl

```diff
@@ -0,0 +1,77 @@
+-name = Configurateur de clavier
+
+app-about = À propos du {-name}
+app-title = System76 {-name}
+
+board-fake = {$model}, fausse
+
+button-cancel = Annuler
+button-configure = Configurer un clavier
+button-disable = Déactiver
+button-import = Importer
+button-test = Vérifier
+
+error-disable-key = Échec dans la désactivation de la clé.
+error-export-keymap = Échec d'exporter le plan de codage.
+error-import-keymap = Échec d'importer le plan de codage.
+error-key-led = Échec en établissant la DEL de la clé.
+error-open-file = Échec dans la lecture du fichier.
+error-save-leds = Échec de sauvegarder les DELs.
+error-set-keyboard-brightness = Échec en établissant la luminosité du clavier.
+error-set-keyboard-mode = Échec en établissant le mode du clavier.
+error-set-keymap = Échec en établissant le plan de codage.
+error-set-layer-brightness = Échec en établissant la luminosité de la couche.
+error-set-layer-color = Échec en établissant la couleur de la couche.
+error-set-layer-mode = Échec en établissant le mode de la couche.
+error-unsupported-keymap = Fichier contenant le plan de codage inacceptable.
+error-unsupported-keymap-desc = Fichier contenant le plan de codage provient d'une version plus récente du logiciel.
+
+key-color = Couleur de la clé:
+
+keymap-for-board = Plan de codage pour '{$model}'
+
+layer-all-brightness = Luminosité (toutes les couches):
+layer-animation-speed = Vitesse de l'animation de la couche:
+layer-color = Couleur de la couche:
+layer-color-pattern = Patron de couleur de la couche:
+layer-saturation = Saturation de la couche:
+
+layout-export = Exporter la disposition
```

Comment:
Here, I feel like "configuration" fits better than "disposition" so it will be "Exporter la configuration"